tripgo.swagger.yaml

```diff
@@ -90,6 +90,110 @@ paths:
+            required:
+              - region
+            example:
+              - region: AU_NSW_Sydney
+                modes:
+                  - pt_pub_tram
+                  - pt_pub_bus
+                operators:
+                  - Metro - Los Angeles
+                routes:
+                  - Metro Blue Line
+                  - 205
+                  - 620
+                  - Metro Purple Line
+      responses:
+        200:
+          description: Successful response
+          schema:
+            type: object
+            properties:
+              code:
+                type: string
+                description: Internal identifier for the region.
+              transitModes:
+                type: array
+                description: Detailed TSPs information per public transit modes.
+                items:
+                  type: object
+                  properties:
+                    identifier:
+                      type: string
+                      description: A specific public transit mode-identifier, e.g., `pt_pub_bus` or `pt_pub_gondola`.
+                    numberOfServices:
+                      type: intiger
+                      description: Total number of services for this region's public transport mode.
+                    realTime:
+                      type: string
+                      description: SHOULD I PUT SOME GLOBAL MODE STATS HERE?
+                    operators:
+                      type: array
+                      description: TSP detailed information.
+                      items:
+                        type: object
+                        properties:
+                          name:
+                              type: string
+                              description: TSP's short name.
+                          numberOfServices:
+                            type: intiger
+                            description: Total number of services for this region's public transport mode and TSP
+                          realTime:
+                            type: string
+                            description: SHOULD I PUT SOME SPECIFIC TSP&MODE STATS HERE?
+                          routes:
+                            type: array
+                            description: Route information
+                            items:
+                              type: object
+                              properties:
+                                name:
+                                  type: string
+                                  description: Route short name
+                                numberOfServices:
+                                  type: intiger
+                                  description: Total number of services for this region's public transport mode, TSP and route
+                                realTime:
+                                  type: string
+                                  description: PUT HERE SPECIFIC RT INFORMATION FOR THIS ROUTE!
```

Comment:
yes.  it is going to be heavy, but that is how i understood the requirement.  would you like to clarify with the customer?